### ISSUE TITLE
Force domain setting from "third party app" pop-up

### ISSUE THREAD

**Issue by Cwpute** [Slion/Fulguris#548]
**Is your feature request related to a problem? Please describe.**
Like reported in this issue https://github.com/Slion/Fulguris/issues/542 some users might see the "third-party app" pop-up show everytime they access a website, which can get annoying.
![Screenshot_20230910-113128](https://github.com/Slion/Fulguris/assets/80604512/9237d40f-73d6-420b-aeb2-2037f0ce53da)
Many users won't know about the domain settings unless they know what they are searching for, so for many this pop-up is their first and only interaction with those settings.

**Describe the solution you'd like**
In that pop-up there could be a checkbox titled "Always for that domain" which would essentialy force the chosen behavior to always be preferred for that domain, and the pop-up to never appear again as a result.

**Comment by Slion** [comment]
Yeah that could be a solution. Though we should really investigate why some users have that happening for every web sites. Possibly has to do with the kind of applications they have installed. 

On my Huawei P30 Pro Android 10 it is happening for every page I load from [AVS Forum](http://avsforum.com) for some reason and then it just wants to open a browser anyway.

**Comment by Slion** [comment]
I just reworked SSL error handling and moved it to domain settings as part of 4a4774c. It has such an option to persist user choice to domain settings.

![image](https://github.com/Slion/Fulguris/assets/6508892/0b9c7247-e682-4719-ad74-4268603c2735)
